src/data/recipes.js

```diff
@@ -56,6 +56,20 @@ export const butter = itemify({
   recipeType: recipeType.KITCHEN,
 })
 
+/**
+ * @property farmhand.module:recipes.oliveOil
+ * @type {farmhand.recipe}
+ */
+export const oliveOil = itemify({
+  id: 'olive-oil',
+  name: 'Olive Oil',
+  ingredients: {
+    [items.olive.id]: 15,
+  },
+  condition: state => state.itemsSold[items.olive.id] >= 150,
```

Comment:
If we significantly increase the amount of olives need to produce oil, we should scale up the condition logic to be consistent with the other recipes. Perhaps requiring 400-500 olives to be sold?

Reply:
yep that makes sense to me!